src/Infrastructure/Repository/BilingualRepository.php

```diff
@@ -125,13 +125,16 @@ final protected function bilingualOption(string $key): MultilingualString
         );
     }
 
-    final protected function bilingualLinkFromTerm(?WP_Term $term): MultilingualLink
+    final protected function bilingualLinkFromTerm(?WP_Term $term, string $taxonomy): MultilingualLink
```

Comment:
I think passing taxonomy is unnecessary because we don't use terms shared between different taxonomies. We haven't used it before for degree and admission requirements, but it might increase performance.